src/standalone/Vegetation/radiation.jl

```diff
@@ -14,8 +14,10 @@ $(DocStringExtensions.FIELDS)
 struct BeerLambertParameters{FT <: AbstractFloat}
     "Leaf angle distribution function (unitless)"
     ld::FT
-    "PAR canopy reflectance (unitless)"
+    "PAR leaf reflectance (unitless)"
```

Comment:
could we use alpha for both? or rho for both? and is rho the same as alpha PAR?

Reply:
Yes sorry we can and should do that, I had changed that originally but then when I reverted to get rid of the SAI I had added I forgot to re-change it whoops